bot/seasons/season.py

```diff
@@ -38,12 +38,19 @@ def get_season(bot, season_name: str = None, date: datetime.date = None):
 
     seasons = get_seasons()
 
-    # If there's a season name, we can just grab the correct class.
+    # Use season override if season name not provided
+    if not season_name and Client.season_override:
+        log.debug(f"Season override found: {Client.season_override}")
+        season_name = Client.season_override
+
+    # Check if a valid season name was provided
     if season_name:
         season_name = season_name.lower()
         if season_name not in seasons:
             season_name = 'evergreen'
 
+    # If there's a season name, we can just grab the correct class.
+    if season_name:
         season_lib = importlib.import_module(f'bot.seasons.{season_name}')
         season_class = getattr(season_lib, season_name.capitalize())
```

Comment:
Should this be put in a utility function? Same thing is done three times in this function.

Reply:
Good idea, I'd edited this part so much that I didn't even think of it. Thanks.